src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBulkRecordReader.java

```diff
@@ -73,6 +90,10 @@ public SalesforceBulkRecordReader(Schema schema) {
     this.jobId = jobId;
     this.batchId = batchId;
     this.resultIds = resultIds;
+    initialRetryDuration = SalesforceSourceConstants.DEFAULT_INITIAL_RETRY_DURATION_SECONDS;
```

Comment:
remove this code, instead use like this in initialize method.
conf.get(SalesforceSourceConstants.CONFIG_INITIAL_RETRY_DURATION, SalesforceSourceConstants.DEFAULT_INITIAL_RETRY_DURATION_SECONDS)

Reply:
this is used for testing purpose only.

Reply:
still it is not solving any purpose, if those properties are null in config, these value will be overidden by null

Reply:
It will require extra mocking for conf and stubs for all the values required for the test cases.